libraries/js/effekt_runtime.js

```diff
@@ -126,9 +126,24 @@ function SHIFT(p, body, ks, k) {
     return () => c(meta, k2)
   }
 
-  let k1 = meta.stack
+  const k1 = meta.stack
   meta.stack = null
-  return body(resumeComp, meta, k1)
+  const resume = (c, ks, k) => RESUME(c, cont, ks, k)
+  return body(resume, meta, k1)
```

Comment:
Do you plan to change this to: 

```
body(c, meta, k1)
```

?

Reply:
No, I think it would not work if I did so.